core/templates/dev/head/pages/topic_editor/subtopics_editor/SubtopicsListTabDirective.js

```diff
@@ -193,10 +204,21 @@ oppia.directive('subtopicsListTab', [
                 '$scope', '$uibModalInstance',
                 function($scope, $uibModalInstance) {
                   $scope.subtopicTitle = '';
+                  $scope.subtopicTitles = subtopicTitles;
+                  $scope.errorMsg = null;
+
+                  $scope.resetErrorMsg = function() {
+                    $scope.errorMsg = null;
+                  };
                   $scope.isSubtopicTitleEmpty = function(subtopicTitle) {
                     return (subtopicTitle === '');
                   };
                   $scope.save = function(title) {
+                    if ($scope.subtopicTitles.indexOf(title) !== -1) {
+                      $scope.errorMsg =
+                        'Subtopic with the title already exists';
```

Comment:
A subtopic with this title already exists.

Reply:
Done.